evm/src/all_stark.rs

```diff
@@ -146,31 +236,79 @@ mod tests {
         keccak_keccak_input_output.extend(Column::singles(
             (0..2 * NUM_INPUTS).map(keccak::registers::reg_output_limb),
         ));
-        let cross_table_lookups = vec![CrossTableLookup::new(
-            vec![TableWithColumns::new(
-                Table::Cpu,
-                Column::singles(cpu_keccak_input_output).collect(),
-                Some(Column::single(cpu::columns::IS_KECCAK)),
-            )],
-            TableWithColumns::new(
-                Table::Keccak,
-                keccak_keccak_input_output,
-                Some(Column::single(keccak::registers::reg_step(NUM_ROUNDS - 1))),
+
+        let cpu_logic_input_output = {
+            let mut res = vec![
+                cpu::columns::IS_AND,
+                cpu::columns::IS_OR,
+                cpu::columns::IS_XOR,
+            ];
+            res.extend(cpu::columns::LOGIC_INPUT0);
+            res.extend(cpu::columns::LOGIC_INPUT1);
+            res.extend(cpu::columns::LOGIC_OUTPUT);
+            res
+        };
+        let logic_logic_input_output = {
+            let mut res = vec![
+                logic::columns::IS_AND,
+                logic::columns::IS_OR,
+                logic::columns::IS_XOR,
+            ];
+            res.extend(logic::columns::INPUT0_PACKED);
+            res.extend(logic::columns::INPUT1_PACKED);
+            res.extend(logic::columns::RESULT);
+            res
+        };
```

Comment:
Maybe we should have these be constructed in the relevant `columns` module, like `cpu::columns::logic_input_output()`. Also goes for the Keccak stuff, so if you want I can also do it in a later PR.

Reply:
Good idea! I’m happy to do it for both Keccak and logic—should be quick :)